src/index.js

```diff
@@ -70,6 +69,33 @@ export default function({ types: t }) {
       },
     },
   };
+  function getFileName( file, options ) {
```

Comment:
I would write a specific test on how this function works. Should be good since it is a pure function, it will need to be exported. 

I would like to see end to end tests also, but I am not sure how to get path data passed in to our tests. I will research that.